README.md

```diff
@@ -143,69 +264,114 @@ Implementing push notifications requires a few additional snippets of code to en
 	    }
 
 	    // Enable or disable features based on the authorization status.
+	    // if you didn't register for remote notifications above you can call `registerForRemoteNotifications` here
+	    // Klaviyo SDK will automatically update the authorization status on next app launch
 	}
 
-	UIApplication.shared.registerForRemoteNotifications()
+    return true
+}
 ```
 
-2. Add the following code to the application delegate file in `application:didRegisterForRemoteNotificationsWithDeviceToken`. You may need to add this code to your application delegate if you have not done so already.
+### Receiving Push Notifications
+
+#### Tracking Opens
 
+Implement the [`userNotificationCenter:didReceive:withCompletionHandler`](https://developer.apple.com/documentation/usernotifications/unusernotificationcenterdelegate/1649501-usernotificationcenter)
+and [`userNotificationCenter:willPresent:withCompletionHandler`](https://developer.apple.com/documentation/usernotifications/unusernotificationcenterdelegate/1649518-usernotificationcenter) to handle push notifications received whe the app is in the background and foreground.
```

Comment:
It looks like the "n" is missing from "received when"

Reply:
```suggestion
and [`userNotificationCenter:willPresent:withCompletionHandler`](https://developer.apple.com/documentation/usernotifications/unusernotificationcenterdelegate/1649518-usernotificationcenter) to handle push notifications received when the app is in the background and foreground.
```

Reply:
I already made the change. Thanks, Evan.